jazz-build-module/events-module.groovy

```diff
@@ -0,0 +1,413 @@
+#!groovy?
+import groovy.json.JsonSlurper
+import groovy.json.JsonOutput
+import groovy.transform.Field
+echo "Events module loaded successfully"
+
+/**
+ * The Events module for jenkins packs
+ * @author: 
+ * @date: 
+*/
+
+@Field def g_request_id = ""
+@Field def g_service = ""
+@Field def g_branch = ""
+@Field def g_runtime = ""
+@Field def g_environment = ""
+@Field def g_region = ""
+@Field def g_domain = ""
+@Field def g_role = ""
+@Field def g_service_type = ""
+@Field def g_event_handler = ""
+@Field def g_event_type = ""
+
+/**
+ * These are the list of available event_names. Any new event names added here should be added to the events table in dynamoDB as well. 
+ */
+/**@Field def Event_Name =[
+	'GENERATE_API_DOC':'GENERATE_API_DOC', 
+	'DELETE_API_DOC':'DELETE_API_DOC', 
+	'UPLOAD_API_SPEC':'UPLOAD_API_SPEC', 
+	'DISABLE_API_CACHE':'DISABLE_API_CACHE', 
+	'UPDATE_SWAGGER':'UPDATE_SWAGGER', 
+	'DEPLOY_SWAGGER_TO_APIGATEWAY':'DEPLOY_SWAGGER_TO_APIGATEWAY', 
+	'UPDATE_DEPLOYMENT_CONF':'UPDATE_DEPLOYMENT_CONF', 
+	'UPDATE_LOGGING_CONF':'UPDATE_LOGGING_CONF', 
+	'GET_DEPLOYMENT_CONF':'GET_DEPLOYMENT_CONF', 
+	'GET_SERVERLESS_CONF':'GET_SERVERLESS_CONF', 
+	'GET_SERVICE_CODE':'GET_SERVICE_CODE', 
+	'MODIFY_TEMPLATE':'MODIFY_TEMPLATE',
+	'CALL_ONBOARDING_WORKFLOW':'CALL_ONBOARDING_WORKFLOW',
+	'RAISE_PR':'RAISE_PR',
+	'VALIDATE_PRE_BUILD_CONF':'VALIDATE_PRE_BUILD_CONF',
+	'CALL_ONBOARDING_SERVICE':'CALL_ONBOARDING_SERVICE',
+	'ADD_WRITE_PERMISSIONS_TO_SERVICE_REPO':'ADD_WRITE_PERMISSIONS_TO_SERVICE_REPO',
+	'CREATE_SERVICE':'CREATE_SERVICE',
+	'BUILD':'BUILD',
+	'CODE_QUALITY_CHECK':'CODE_QUALITY_CHECK',
+	'UNIT_TEST':'UNIT_TEST',
+	'COMMIT_CODE':'COMMIT_CODE',
+	'APPROVE_PR':'APPROVE_PR',
+	'CREATE_SERVICE_REPO':'CREATE_SERVICE_REPO',
+	'LOCK_MASTER_BRANCH':'LOCK_MASTER_BRANCH',
+	'DEPLOY_TO_AWS':'DEPLOY_TO_AWS',
+	'UNDEPLOY_WEBSITE':'UNDEPLOY_WEBSITE',
+	'PUSH_TEMPLATE_TO_SERVICE_REPO':'PUSH_TEMPLATE_TO_SERVICE_REPO',
+	'CLONE_TEMPLATE':'CLONE_TEMPLATE',
+	'CREATE_ASSET':'CREATE_ASSET',
+	'UPDATE_ASSET':'UPDATE_ASSET',
+	'DELETE_API_RESOURCE':'DELETE_API_RESOURCE',
+	'DELETE_CLOUDFRONT':'DELETE_CLOUDFRONT',
+	'DISABLE_CLOUDFRONT':'DISABLE_CLOUDFRONT',
+	'DELETE_S3BUCKET_POLICY':'DELETE_S3BUCKET_POLICY',
+	'BACKUP_PROJECT':'BACKUP_PROJECT',
+	'DELETE_PROJECT':'DELETE_PROJECT',
+	'ENABLE_AD_AUTHORIZATION':'ENABLE_AD_AUTHORIZATION',
+	'CREATE_DEPLOYMENT': 'CREATE_DEPLOYMENT',
+	'UPDATE_DEPLOYMENT': 'UPDATE_DEPLOYMENT',
+	'UPDATE_ENVIRONMENT': 'UPDATE_ENVIRONMENT',
+	'DELETE_DEPLOYMENT':'DELETE_DEPLOYMENT'
+
+]*/
+
+@Field def Event_Names = [
+	'VALIDATE_INPUT': 'VALIDATE_INPUT',
```

Comment:
Use similar enumeration in the other Jenkins files too.

Reply:
I will use that . I will be removing this file as this is been checked in  Part 2 which is done by Sini